src/main/flight/servos.c

```diff
@@ -418,6 +418,12 @@ void servoMixer(void)
 
     for (int i = 0; i < MAX_SUPPORTED_SERVOS; i++) {
         servo[i] = ((int32_t)servoParams(i)->rate * servo[i]) / 100L;
+        if (servoConfig()->aileron_differential && !FLIGHT_MODE(FLAPERON)) {
```

Comment:
I think this should be done differently - by differently weights for positive and negative values of `INPUT_STABILIZED_ROLL` in servo mixer.